src/workspace/views/QuickLaneOverview.tsx

```diff
@@ -255,86 +283,33 @@ const QuickLaneOverview: FunctionComponent<QuickLaneOverviewProps> = ({
 		}
 	};
 
-	const renderHeader = () => (
-		<Toolbar>
-			<ToolbarLeft></ToolbarLeft>
-			<ToolbarRight>
-				<ToolbarItem>
-					<Form type="inline">
-						<FormGroup>
-							<TextInput
-								className="c-assignment-overview__search-input"
-								icon="filter"
-								value={filterString}
-								onChange={setFilterString}
-								disabled={quickLanes.length <= 0}
-							/>
-						</FormGroup>
-					</Form>
-				</ToolbarItem>
-			</ToolbarRight>
-		</Toolbar>
-	);
-
-	const renderTable = () => (
-		<>
-			{renderHeader()}
-			<Table
-				columns={columns}
-				data={paginated}
-				emptyStateMessage={t(
-					'workspace/views/quick-lane-overview___je-hebt-nog-geen-gedeelde-links-aangemaakt'
-				)}
-				renderCell={renderCell}
-				rowKey="id"
-				variant="styled"
-				onColumnClick={onClickColumn as any}
-				sortColumn={sortColumn}
-				sortOrder={sortOrder}
-			/>
-			<Spacer margin="top-large">
-				<Pagination
-					pageCount={Math.ceil(numberOfItems / ITEMS_PER_PAGE)}
-					currentPage={page}
-					onPageChange={setPage}
-				/>
-			</Spacer>
-		</>
-	);
-
-	const renderEmptyFallback = () => (
-		<ErrorView
-			icon="link-2"
-			message={t(
-				'workspace/views/quick-lane-overview___je-hebt-nog-geen-gedeelde-links-aangemaakt'
-			)}
-		>
-			<p>
-				{t(
-					'workspace/views/quick-lane-overview___een-gedeelde-link-kan-je-gebruiken-om-makkelijk-en-snel-fragmenten-of-collecties-te-delen-zonder-tijdslimiet'
-				)}
-			</p>
-			<Spacer margin="top">
-				<Button
-					type="primary"
-					icon="search"
-					label={t('workspace/views/quick-lane-overview___zoek-een-item')}
-					title={t(
-						'workspace/views/quick-lane-overview___zoek-een-item-en-maak-een-gedeelde-link-er-naartoe'
-					)}
-					onClick={() => history.push(APP_PATH.SEARCH.route)}
-				/>
-			</Spacer>
-		</ErrorView>
-	);
-
-	const renderQuickLanes = () => (quickLanes.length > 0 ? renderTable() : renderEmptyFallback());
-
 	return (
-		<LoadingErrorLoadedComponent
-			loadingInfo={loadingInfo}
-			dataObject={quickLanes}
-			render={renderQuickLanes}
+		<FilterTable
+			columns={columns}
+			data={quickLanes}
+			dataCount={quickLanes.length}
+			itemsPerPage={ITEMS_PER_PAGE}
+			noContentMatchingFiltersMessage={
+				loadingInfo.state === 'loaded'
+					? t(
+							'workspace/views/quick-lane-overview___er-werden-geen-gedeelde-links-gevonden-die-voldoen-aan-de-opgegeven-criteria'
+					  )
+					: ''
+			}
+			onTableStateChanged={(state) => {
+				// NOTE: prevents recursion loop but hits theoretical performance
+				if (JSON.stringify(filters) !== JSON.stringify(state)) {
```

Comment:
JS garandeert niet dat keys altijd in de juiste volgorde staan (tenzij je weet dat het hier niet voorvalt)
Lodash heeft een `isEqual` function of liep je hier tegen performance issues aan?